### DISCUSSION TITLE
KFG Community Practice Call

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#118] [Ideas]
Ideas for intentional practices to do in community for our weekly call.

**Comment by seekanddefine** [comment]


**Reply by seekanddefine** [reply]
Need to figure out line wrapping lol. Still better than my last chart attempt!

**Reply by avidrucker** [reply]
I like your effort! Does [this spreadsheet](https://docs.google.com/spreadsheets/d/1TOVDz9JBn0txkB4ISdCts1e8KCH2v5dYcZQwNcaPsrk/edit?usp=sharing) bring you any ease or accessibility?

**Comment by seekanddefine** [comment]
Should we stick with 60 min or consider 75 as an option? I feel like it is rushed pushing it into 60.

**Reply by avidrucker** [reply]
@seekanddefine I'd lean towards a `60 min+` and suggest to `split-this`, because I think that micro-tasking will help us stay focused and accountable. In other words, a "75 min" task would become a "60 min" plus a "15 min".

**Reply by avidrucker** [reply]
I'm sorry, I see now that I misunderstood your question @seekanddefine 😅 (I thought you were referring to issue labels) This is clearly referring to the community call, right?

75 min sounds good to me 👍😊

**Comment by seekanddefine** [comment]
[Community Practice 1.md](https://github.com/knowledge-for-good/knowledgeforgood/files/6948277/Community.Practice.1.md)

If it's not legible, people aren't going to read it. Instead of editing it I'm just going to leave this here.
